location/templates/location.html

```diff
@@ -177,7 +178,7 @@ <h4 class=" mb-0">
 {% if messages %}
```

Comment:
commented out these lines since I thought the favorite button served the purpose of showing if the favorite/unfavorite function was implemented but this comes in handy when contacting the landlord since we currently don't inform the user that an email was sent